roofit/roofitcore/inc/NumericalDerivatorMinuit2.h

```diff
@@ -0,0 +1,117 @@
+// @(#)root/mathcore:$Id$
+// Authors: L. Moneta, J.T. Offermann, E.G.P. Bos    2013-2018
+//
+/**********************************************************************
+ *                                                                    *
+ * Copyright (c) 2013 , LCG ROOT MathLib Team                         *
+ *                                                                    *
+ **********************************************************************/
+/*
+ * NumericalDerivatorMinuit2.h
+ *
+ *  Original version (NumericalDerivator) created on: Aug 14, 2013
+ *      Authors: L. Moneta, J. T. Offermann
+ *  Modified version (NumericalDerivatorMinuit2) created on: Sep 27, 2017
+ *      Author: E. G. P. Bos
+ */
+
+#ifndef RooFit_NumericalDerivatorMinuit2
+#define RooFit_NumericalDerivatorMinuit2
+
+#ifndef ROOT_Math_IFunctionfwd
+#include <Math/IFunctionfwd.h>
+#endif
+
+#include <vector>
```

Comment:
It's a bit nicer if `<vector>` as well as other standard headers comes last, so you would see if one of those headers had forgotten to include it.